agent/php_execute.c

```diff
@@ -2071,13 +1876,7 @@ static void nr_php_instrument_func_begin(NR_EXECUTE_PROTO) {
     nr_php_observer_attempt_call_cufa_handler(NR_EXECUTE_ORIG_ARGS);
   }
   wraprec = nr_php_get_wraprec(execute_data->func);
-  /*
-   * If there is custom instrumentation or tt detail is more than 0, start the
-   * segment.
-   */
-  if ((NULL == wraprec) && !(NRINI(tt_detail) && NR_OP_ARRAY->function_name)) {
-    return;
-  }
+
   /*
    * Check if it's a custom error handler.  Even with some custom error
    * handlers, fcall might not get called. But we don't need to wait for
```

Comment:
This isn't true anymore,right?

Reply:
Fixed the language in [bce3e14](https://github.com/newrelic/newrelic-php-agent/pull/767/commits/bce3e1462db26bb5f841542c199dae92ba96a1b3)